sources/documents/RTB/Verbali/Interni/VerbaleInterno_2024-05-02/capitolo-riunione.tex

```diff
@@ -0,0 +1,73 @@
+\subsubsection{Amministratore}
+\par Vengono discussi i nuovi comandi per la citazione dei documenti in \glossario{LateX}, inseriti nel documento di \NdP. Sono anche avanzati i lavori sul \PdP\ e iniziati i lavori sul \PdQ. Sono anche stati sviluppati tabelle e grafici da inserire per preventivi e consuntivi del \PdP\ e sono iniziati i lavori sui file di configurazione per la compilazione dei file pdf in au1tomatico.
+\subsubsection{Responsabile} 
+\par Il Responsabile si è occupato della pianificazione del preventivo del primo \glossario{sprint}\ e l'elaborazione di tabelle e grafici per preventivo e consuntivo degli \glossario{sprint}. Sono inoltre stati definiti attività e grafico \glossario{Gannt}\ su \glossario{Jira}\ e steso il verbale del meeting precedente.
+\subsubsection{Programmatore}
+\par Il Programmatore discute dello studio e dei progressi ottenuti con \glossario{txtai}, spiegando al gruppo la trasformazione che la tecnologia opera al fine di trasformare il linguaggio naturale in un vettore e in un indice di vettori. L'idea per il funzionamento è quella di formattare dizionario dati, \glossario{query}\ e le informazioni delle tabelle in documenti che simulino il \glossario{database}. La \glossario{query} verrebbe pre-processata e generare dei \glossario{token} da questa, per renderla più accessibile ai meccanismi di \glossario{txtai}. La frase verrebbe quindi processata e passata all'indicizzatore che eseguirebbe lo stesso procedimento con il dizionario dati. Rispetto ai lavori precedenti sono stati aggiunti due nuovi moduli per la pre-processazione.
+\par Vengono infine discussi i prossimi lavori che riguardano la creazione di un secondo indice che faccia da indice intermedio tra la \glossario{query}\ e l'indice reale per estrarre l'informazione. Questo sarebbe utile perché arricchirebbe i risultati parziali con informazioni quali la descrizione della tabella e i vincoli comuni. Oltre a ciò c'è la volontà di testare l'inserimento del numero di colonne per confrontare e capire se il risultato finale sia ottimale.
+\subsubsection{Progettista}
+\par Il Progettista ha lavorato coordinandosi con il programmatore per la strutturazione di un dizionario dati in formato \glossario{JSON}\ che potesse essere più efficiente nell'interazione con \glossario{ChatGPT}. È stata esposta poi la presenza di due tipi di formato in linguaggio naturale che più sembrano essere ottimali per gli \glossario{LLM}. Infine viene proposto di utilizzare il linguaggio \glossario{Python}\ per il \glossario{backend}\ vista la quantità già scritta di codice per la parte funzionale del progetto. 
```

Comment:
- amplierei con la citazione dei formati analizzati 

È stata esposta poi la presenza di due tipi di formati in linguaggio naturale che sembrano essere più adatti per gli \glossario{LLM}.

Reply:
```suggestion
\par Il Progettista ha lavorato coordinandosi con il programmatore per la strutturazione di un dizionario dati in formato \glossario{JSON}\ che potesse essere più efficiente nell'interazione con \glossario{ChatGPT}. Sono stati citati diversi tipi di formato in linguaggio naturale che risultano essere più adatti gli \glossario{LLM} che verranno approfonditi in seguito. Infine è stato proposto di utilizzare il linguaggio \glossario{Python}\ per il \glossario{backend}\ vista la quantità già scritta di codice per la parte funzionale del progetto. 
```